lib/rtcpeerconnection/chrome.js

```diff
@@ -117,12 +124,41 @@ function ChromeRTCPeerConnection(configuration) {
 
 inherits(ChromeRTCPeerConnection, EventTarget);
 
-ChromeRTCPeerConnection.prototype.addStream = function addStream(stream) {
-  if (this._localStreams.has(stream)) {
-    return;
+ChromeRTCPeerConnection.prototype.addTrack = function addTrack() {
+  var args = [].slice.call(arguments);
+  if ('addTrack' in PeerConnection.prototype) {
+    return this._peerConnection.addTrack.apply(this._peerConnection, args);
+  }
+  var mediaStreamTrack = args[0];
+  var syncMediaStreams = args.slice(1);
```

Comment:
In the spec, these are called `track` and `streams`. Can we go with the terser names?